src/features/fireWeather/pages/MoreCastPage.test.tsx

```diff
@@ -189,24 +194,9 @@ it('renders daily model, forecast, and hourly values in response to user inputs'
     getByTestId('temp-rh-graph'),
     getByTestId('wx-graph-reading-toggle'),
     getByTestId('wx-graph-model-toggle'),
-    getByTestId('wx-graph-model-summary-toggle'),
-    getByTestId('wx-graph-forecast-summary-toggle'),
     getByTestId('wx-graph-bias-toggle')
   ])
 
-  // Check to see if some of SVG are rendered in the graph (dots, area, and tooltip)
```

Comment:
Do we no longer have a mechanism to test that items are being rendered?

Reply:
We are testing these more thoroughly in Cypress, so I considered it as duplicate effort